pkg/util/mount/mount.go

```diff
@@ -43,6 +46,9 @@ type Interface interface {
 	DeviceOpened(pathname string) (bool, error)
 	// PathIsDevice determines if a path is a device.
 	PathIsDevice(pathname string) (bool, error)
+	// GetVolumeIDFromMount finds the volume id from its mount path which
```

Comment:
We should define exactly what a volume ID is here, and at least know in our heads (if not reflect in comment) where this is used and why it is important.